torch/csrc/jit/export.cpp

```diff
@@ -38,11 +33,13 @@ struct ExportContext {
 
 void encodeGraph(onnx::GraphProto * p_g, const std::shared_ptr<Graph> & g,
                  const std::vector<at::Tensor> & initializers,
-                 ExportContext *ctx, RawDataExportMap* raw_data_export_map=nullptr);
+                 ExportContext *ctx, RawDataExportMap* raw_data_export_map=nullptr,
+                 bool export_raw_ir = false);
```

Comment:
What does `export_raw_ir` mean?

Reply:
Perhaps this parameter should go in ExportContext? It seems to be threaded around everywhere.

Reply:
+1 for this

Reply:
Good point, I'll move it to ExportContext and document it.